From b0b2a4380846270b116571bf87dd7c77eb0a9ca7 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Wed, 2 Sep 2015 19:07:01 -0400
Subject: [PATCH 03/14] add persist.privacy. property namespace

This will be used to control whether the MAC address is randomized,
among other things.
---
 property_contexts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/property_contexts b/property_contexts
index bc41b54..2837486 100644
--- a/property_contexts
+++ b/property_contexts
@@ -94,3 +94,4 @@ dalvik.                 u:object_r:dalvik_prop:s0
 ro.dalvik.              u:object_r:dalvik_prop:s0
 
 persist.libc.malloc.    u:object_r:system_prop:s0
+persist.privacy.        u:object_r:system_prop:s0
-- 
2.10.0

